CanDIG/htsget_app
# DIG-1308 Switch HTSGet to Postgres

# Jira tickets
[DIG-1308](https://candig.atlassian.net/browse/DIG-1308)
[DIG-1309](https://candig.atlassian.net/browse/DIG-1309)

# Description
This PR switches HTSGet from using a sqlite backend to Postgres, This also renames the `ncbiRefSeq` table to `ncbirefseq`, as the backend requires it, and its value of `end` to `endpos` (internally only), as `end` is a keyword in Postgres.

# Related
This requires a change to CanDIGv2 as well, in [this PR](https://github.com/CanDIG/CanDIGv2/pull/296)

# To test:
- Should just run integration-tests I think.

[DIG-1308]: https://candig.atlassian.net/browse/DIG-1308?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DIG-1309]: https://candig.atlassian.net/browse/DIG-1309?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
